src/sass/_shame-wizard.scss

```diff
@@ -0,0 +1,164 @@
+// This file is a place where you put all the code you're not proud of,
+// with the intention of A) isolating it and B) fixing it later. The goal
+// is for this file to be empty, but some times you just have to throw that
+// hack in there to get it working.
+
+// message while loading react app
+.loading-message {
```

Comment:
I just did a quick diff between _shame-wizard.scss and _shame.scss. The only difference is _shame.scss also has  the expander stuff used in the current apply page. We can just delete the shame wizard and stick with _shame. (We're currently not even using this file.)